core/blockly.js

```diff
@@ -655,20 +656,9 @@ const bindEventWithChecks_ = function(
 exports.bindEventWithChecks_ = bindEventWithChecks_;
 
 // Aliases to allow external code to access these values for legacy reasons.
-exports.DRAG_RADIUS = internalConstants.DRAG_RADIUS;
-exports.FLYOUT_DRAG_RADIUS = internalConstants.FLYOUT_DRAG_RADIUS;
-exports.SNAP_RADIUS = internalConstants.SNAP_RADIUS;
-exports.CONNECTING_SNAP_RADIUS = internalConstants.CONNECTING_SNAP_RADIUS;
-exports.CURRENT_CONNECTION_PREFERENCE =
-    internalConstants.CURRENT_CONNECTION_PREFERENCE;
-exports.BUMP_DELAY = internalConstants.BUMP_DELAY;
 exports.COLLAPSE_CHARS = internalConstants.COLLAPSE_CHARS;
 exports.DRAG_STACK = internalConstants.DRAG_STACK;
-exports.SPRITE = internalConstants.SPRITE;
-exports.DRAG_NONE = internalConstants.DRAG_NONE;
-exports.DRAG_STICKY = internalConstants.DRAG_STICKY;
-exports.DRAG_BEGIN = internalConstants.DRAG_BEGIN;
-exports.DRAG_FREE = internalConstants.DRAG_FREE;
+exports.SPRITE = constants.SPRITE;
```

Comment:
Should this export be marked as deprecated, or deleted outright?

The entry in `renamings.js` suggests that we expect developers to use `Blockly.constants.SPRITE` rather than `Blockly.SPRITE`.  I would not suggest creating a `set` accessor just to print a deprecation warning, but perhaps putting a comment about it so we remember to delete it in due course, if you don't want to do so now?

Reply:
Hmm, I think it is actually fine to delete now. I looked back through GitHub and didn't see anyone using it externally. 

This makes me think though- do we actually want to keep `internal_constants.js` around for values like this? I think the original issue says we should remove all values from internal_constants, but I think it serves a different purpose than `constants` which are public and we expect developers to use.

Reply:
That seems like a good idea to me: don't make public anything which doesn't need to be.  An alternative might be to introduce a `core/sprite.js` (`Blockly.sprite` perhaps?) that is imported by the two modules which need these constants.  (I had a look to see if there were any other shared dependencies which might be a suitable home; the only one that looked at all promising was `core/positionable_helpers.js` (`Blockly.uiPosition`), but that doesn't seem quite right.

Maybe @rachel-fenichel has an opinion about this?

Reply:
I'm fine with deleting it from exports now and eventually moving it to its own file (core/sprite.js is reasonable). We certainly don't need to keep internal constants around long-term--everything in it should eventually end up in a more logical place.